src/test/java/com/puppycrawl/tools/checkstyle/AbstractXmlTestSupport.java

```diff
@@ -60,6 +60,10 @@ protected static void verifyXml(String expectedOutputFile,
                 expectedContents);
         final Document actualDocument = getOutputStreamXml(actualOutputStream);
 
+        Assert.assertEquals("xml encoding should be the same: ", expectedDocument.getXmlEncoding(),
```

Comment:
Do we really need the `: ` on the end?
Most assertion text don't end this way.

Reply:
Please see the whole file `AbstractXmlTestSupport.java`:
https://github.com/Softus/checkstyle/blob/305722f2e3f42c3a96bb3b182d5255bae7dea53b/src/test/java/com/puppycrawl/tools/checkstyle/AbstractXmlTestSupport.java

I just follow the style used by all the other checks in this file. If it is wrong, then the entire file should be rewritten.

Reply:
Sorry, my bad. The rest is in the form `"should be: " + path`. The colon is removed.